demos/twa-basic/src/main/AndroidManifest.xml

```diff
@@ -49,6 +50,10 @@
             <meta-data android:name="android.support.customtabs.trusted.DISPLAY_MODE"
                 android:value="immersive"/>
 
+
+            <meta-data android:name="android.support.customtabs.trusted.SCREEN_ORIENTATION"
+                android:value="landscape"/>
```

Comment:
So I do like the idea of having an example for how to set the value in our basic demo, but I don't think that I want the `twa-basic` demo to launch in landscape. Could you just change this to portrait please?

Reply:
Sure.